plan.go

```diff
@@ -15,7 +15,7 @@ var _ Plans = (*plans)(nil)
 // Plans describes all the plan related methods that the Terraform Enterprise
 // API supports.
 //
-// TFE API docs: https://www.terraform.io/docs/cloud/api/plan.html
+// TFE API docs: https://www.terraform.io/cloud-docs/api-docs/plans
```

Comment:
FYI, this is not suppose to be `plans.html` . This would take you to a "page not found"